readme.md

```diff
@@ -897,6 +897,7 @@
 - [Computational Geometry](https://github.com/atkirtland/awesome-computational-geometry#readme) - Computational approaches for problems in geometry.
 - [OpenStreetMap](https://github.com/osmlab/awesome-openstreetmap#readme) - An open data mapping project utilized by many apps and devices.
 - [Computational Biology](https://github.com/inoue0426/awesome-computational-biology#readme) - Computational approaches applied to problems in biology.
+- [AI Collection](https://github.com/ai-collection/ai-collection) - A Collection of Awesome Generative AI Applications
```

Comment:
You have CI that reports for markdown mistakes in accordance with lychee, but not with awesome-lint. As an extra measure, you should add that as a PR check.

Reply:
Lychee is used to validate that links are not broken. I will add awesome lint as well to verify no duplicated links.